nautobot_ssot/jobs/base.py

```diff
@@ -110,6 +110,18 @@ def sync_data(self):
         - self.sync       (Sync instance tracking this job execution)
         - self.job_result (as per Job API)
         """
+
+        def record_memory_trace(step: str):
+            """Helper function to record memory usage and reset tracemalloc stats."""
+            memory_size, memory_peak = tracemalloc.get_traced_memory()
+            setattr(self.sync, f"{step}_memory_size", memory_size)
+            setattr(self.sync, f"{step}_memory_peak", memory_peak)
+            self.sync.save()
+            self.log_info(
+                message=(f"Traced memory for {step} (Current, Peak): {memory_size} bytes, {memory_peak} bytes")
```

Comment:
Would it be more consistent and correct to log as `(Final, Peak)` instead of `(Current, Peak)`?